metrics/stats.go

```diff
@@ -0,0 +1,161 @@
+package metrics
+
+import (
+	"runtime"
+	"sync"
+	"sync/atomic"
+	"time"
+)
+
+type Stats struct {
```

Comment:
__[golint]__
* [golint] exported type Stats should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->